packages/rancher-prom2teams/generated-changes/overlay/templates/role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "prom2teams.fullname" . }}
```

Comment:
I would make this specifically for the PSP, add `-psp`.

Reply:
fixed.